README.md

```diff
@@ -16,7 +16,7 @@ In this documentation we'll be talking about the different aspects of our techno
 
 ### Introduction
 
-To be able to do interesting analysis of source code, we need to treat our code as data. However, analyzing code as plain text is very limiting since our compilers actually use the [Abstract Syntax Tree](https://en.wikipedia.org/wiki/Abstract_syntax_tree) \(AST\).  
```

Comment:
That was correct

Reply:
In my opinion, it should be either "an interesting analysis" (singular), or "interesting analyses" (plural). But I will change that back if you wish- it conveys the meaning either way.

Reply:
You got a point